src/server.c

```diff
@@ -4575,6 +4576,17 @@ int finishShutdown(void) {
                   num_replicas);
     }
 
+    if (safe && server.cluster_enabled && clusterNodeIsVotingPrimary(getMyClusterNode())) {
+        if (force) {
+            serverLog(LL_WARNING, "I am a voting primary, shutting down may cause the cluster to down. Exit anyway.");
+        } else {
+            serverLog(LL_WARNING, "I am a voting primary, shutting down may cause the cluster to down, can't exit.");
+            if (server.supervised_mode == SUPERVISED_SYSTEMD)
+                serverCommunicateSystemd("I am a voting primary, shutting down may cause the cluster to down, can't exit.\n");
```

Comment:
```suggestion
                serverCommunicateSystemd("I am a voting primary, shutting down may cause the cluster to go down, can't exit.\n");
```

Reply:
@jsoref's suggestion + more

```suggestion
                serverCommunicateSystemd("This is a voting primary. Shutting down may cause the cluster to go down. Can't exit.\n");
```